tests/test_config_flow.py

```diff
@@ -148,6 +152,8 @@ async def test_user_invalid_url(hass: HomeAssistant) -> None:
         result["flow_id"],
         {
             CONF_URL: "THIS IS NOT A URL",
+            CONF_USERNAME: "",
+            CONF_PASSWORD: "",
```

Comment:
As above, lets either test explicit values or None if not being passed into the flow.